nix/overlay.nix

```diff
@@ -57,8 +57,8 @@ in {
   yarn = super.yarn.override { nodejs = super.nodejs-18_x; };
   openjdk = super.openjdk11_headless;
   xcodeWrapper = super.xcodeenv.composeXcodeWrapper {
```

Comment:
This should point to local copy of `xcodewrapper.nix` Still working on resolving a few quirks